Split core get asset details (#40050)
demisto/content: Packs/Core/ReleaseNotes/3_4_0.md

#### Integrations

##### Investigation & Response

- The **core-get-asset-details** command has been moved to the new ***Cortex Platform Core*** integration.

> ##### New: Cortex Platform Core

Reviewer: ```suggestion
##### New: Cortex Platform - Core
```

- New: Added a new integration - ***Cortex Platform Core*** that uses the Cortex API to provide commands for accessing essential services and capabilities of the Cortex Platform.

> - Added the ***core-get-asset-details*** command previously exposed by the **Investigation & Response** integration.

Reviewer: ```suggestion
```
it's all in the same RN so redundant to mention twice

#### Integrations

##### Investigation & Response

> - The ***core-get-asset-details*** command has been deprecated and moved to the new **Cortex Platform Core** integration.
> 
> ##### New: Cortex Platform - Core
> 
> - This integration uses the Cortex API for accessing essential services and capabilities of the Cortex platform.
> - Added the ***core-get-asset-details*** command previously exposed by **Investigation & Response**.
> The command will eventually be removed from **Investigation and Response** completely. Please ensure any playbooks using this command are not referencing the **Investigation and Reponse** integration.

Reviewer: ```suggestion
- The ***core-get-asset-details*** command has been deprecated and moved to the new **Cortex Platform - Core** integration. **Note**: The command will soon be removed from this integration. Please ensure all playbooks that use this command switch to using its version from the **Cortex Platform - Core** integration.

##### New: Cortex Platform - Core

- This integration uses the Cortex API for accessing essential services and capabilities of the Cortex platform.
```
